### ISSUE TITLE
accessibility id is not visible when inspect element

### ISSUE THREAD

**Issue by AlirezaAzizi145** [NativeScript/NativeScript#9579]
Hi, i use Appium in my Nativescript project. i want to find elements by accessibility-id but that's not visible.
i dont want to use xpath instead accessibility-id beacause i have a lot of changes on my UI, so it's not work for me.
There is an interesting point that the accessibility-id is only show when i inspect my Nativescript searchbar

**Comment by rigor789** [comment]
See https://github.com/NativeScript/NativeScript/issues/9450
